packages/framework-core/src/booster-graphql-dispatcher.ts

```diff
@@ -27,9 +26,7 @@ export class BoosterGraphQLDispatcher {
 
   public constructor(private config: BoosterConfig, private logger: Logger) {
     this.readModelDispatcher = new BoosterReadModelDispatcher(config, logger)
```

Comment:
🤔 I see that a similar thing is happening here with the `readModelDispatcher` and later with the `commandDispatcher`. Now that we're not using it to initialize the `GraphQLGenerator`, we're now using two separate instances that are initialized with the same values, would it make sense to make them a singleton class too?

Reply:
I think we can make another PR to refactor more classes into Singleton. I'll create the issue in GitHub